docs/k0s-multi-node.md

```diff
@@ -4,16 +4,17 @@ As k0s binary has everything it needs packaged into a single binary, it makes it
 
 ## Pre-requisites
 
-Download k0s binary from [releases](https://github.com/k0sproject/k0s/releases/latest) and push it to all the nodes you wish to connect to the cluster.
+Install k0s using [instructions](k0s-install.md)
 
-That's it, really.
 
 ## Bootstrapping controller node
 
 Create a [configuration](configuration.md) file if you wish to tune some of the settings.
 
```

Comment:
Worth adding the `k0s default-config` command and then forward users to the configuration page.